### DISCUSSION TITLE
What do you think about raylib?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#148] [Ideas]
Do you think Objeck should have a binding for it?

https://github.com/raysan5/raylib

It's an alternative to SDL2 that become popular just recently. It's used by hobbyists more than SDL2 as it's more simple. The cons of it is it's not battle tested like SDL2 and there is no promise for backward compatibility, the APIs could change anytime if the developer wants. It's the reason why the developer said bindings for raylib should be auto generated. But the APIs of raylib is simple enough that making such a binding generator very easy. Here is one of SmallBASIC:

https://github.com/smallbasic/smallbasic.plugins/tree/master/tools

**Comment by objeck** [comment]
Community?

**Reply by ghost** [reply]
OK. If I can do it. No promise.
